CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.22.0] - 2024-03-15
+
+### Added
+
+### Removed
+
+### Changed
+
+### Fixed
+
```

Comment:
why is changelog empty?

Reply:
We're releasing 3.21.0 - we're creating a new entry for the next release